src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/featuredvideo/FeaturedVideoComponentObject.java

```diff
@@ -0,0 +1,164 @@
+import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+
+import org.openqa.selenium.Cookie;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+public class FeaturedVideoComponentObject extends WikiBasePageObject {
+
+  private static final String AUTOPLAY_COOKIE = "featuredVideoAutoplay";
+
+  @FindBy(css = ".featured-video")
+  private WebElement featuredVideo;
+
+  @FindBy(css = ".wikia-jw-settings-button")
+  private WebElement settingsMenu;
+
+  @FindBy(css = ".jw-title-primary")
+  private WebElement primaryTitle;
+
+  @FindBy(css = ".jw-title-secondary")
+  private WebElement secondaryTitle;
+
+  @FindBy(css = "#featured-video__player")
+  private WebElement player;
+
+  @FindBy(css = ".jw-controlbar")
+  private WebElement controlbar;
+
+  @FindBy(css = ".jw-svg-icon-play")
+  private WebElement controlbarPlayIcon;
+
+  @FindBy(css = ".jw-svg-icon-pause")
+  private WebElement controlbarPauseIcon;
+
+  @FindBy(css = ".video-feedback")
+  private WebElement videoFeedback;
+
+  @FindBy(css = "#featured-video__player-videoAutoplayToggle")
+  private WebElement autoplayToggle;
+
+  @FindBy(css = ".jw-icon-tooltip.jw-icon-volume.jw-off")
+  private WebElement volumeMuted;
+
+  @FindBy(css = ".wikia-jw-settings__quality-button")
+  private WebElement videoQualityButton;
+
+  @FindBy(css = ".wikia-jw-settings__captions-button")
+  private WebElement videoCaptionsButton;
+
+  @FindBy(css = ".wikia-jw-settings__submenu")
+  private List<WebElement> videoSettingsSubmenu;
+
+  public FeaturedVideoComponentObject() {
+  }
+
+  public FeaturedVideoComponentObject setAutoplayCookie(boolean autoplay) {
+    driver.manage().addCookie(new Cookie(
+        AUTOPLAY_COOKIE,
+        autoplay ? "1" : "0",
+        String.format(".%s", Configuration.getEnvType().getWikiaDomain()),
+        null,
+        null
+    ));
+
+    return this;
+  }
+
+  public FeaturedVideoComponentObject openWikiArticle(String articleName) {
+    this.openWikiPage(getWikiUrl() + URLsContent.WIKI_DIR + articleName + "?noads=1");
+
+    return this;
+  }
+
+  public boolean isFeaturedVideo() {
+    wait.forElementVisible(featuredVideo);
+
+    return featuredVideo.isDisplayed();
+  }
+
+  public String getTitle() {
+    wait.forElementVisible(primaryTitle);
+
+    return primaryTitle.getText();
+  }
+
+  public String getSubtitle() {
+    wait.forElementVisible(secondaryTitle);
+
+    return secondaryTitle.getText();
+  }
+
+  public FeaturedVideoComponentObject clickPlay() {
+    wait.forElementClickable(player);
+    player.click();
+
+    return this;
+  }
+
+  public FeaturedVideoComponentObject clickPause() {
+    wait.forElementClickable(player);
+    player.click();
+
+    return this;
+  }
+
+  public boolean isVideoPlaying() {
+    wait.forElementVisible(videoFeedback);
```

Comment:
if you want to check if video is playing I would not rely on video feedback element. There is special css class added to one of divs related to player itself that appears when video is playing